### PR TITLE
Moisture database

### PR DESCRIPTION
**Pull request by laurencetaylor**

### CODE REVIEW

File: src/moisture_reading_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import moisture_reading
+import sqlite3
+from datetime import datetime
+
+conn = sqlite3.connect('soil_moisture_test.db')
```
Comment (breycarr) [inline src/moisture_reading_test.py:5]: Have you tested with the live DB?
Do you need to check that works?

File: src/moisture_reading.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import eel
```
Comment (breycarr) [inline src/moisture_reading.py:1]: Why is eel being imported here when we don't currently use it?

**Review by breycarr — APPROVED** [review]
Looks good, hopefully implementing eel will work well!

File: src/moisture_reading.py (hunk 2 above)
Comment (laurencetaylor) [reply src/moisture_reading.py:1]: We will use it shortly! Could have left it out for now

File: src/moisture_reading_test.py (hunk 1 above)
Comment (laurencetaylor) [reply src/moisture_reading_test.py:5]: Not tested the real DB, should be OK if the test one works but will check

**Pull request merged**